app/src/androidTest/java/com/sdpteam/connectout/remoteStorage/FileStorageFirebaseTest.java

```diff
@@ -38,4 +40,21 @@ public void testUploadImageToFirebase() {
         Uri createdUrl = fJoin(new FileStorageFirebase().uploadFile(uri, "jpg"));
         assertTrue(createdUrl.toString().contains("https://"));
     }
+
+    @Test
+    public void testUploadImagesToFirebase() {
+        Resources resources = intentsTestRule.getActivity().getResources();
+
+        // Create the resource Uri for the event_image drawable
+        Uri uri = Uri.parse(ContentResolver.SCHEME_ANDROID_RESOURCE + "://" +
+                resources.getResourcePackageName(R.drawable.event_image) + '/' +
+                resources.getResourceTypeName(R.drawable.event_image) + '/' +
+                resources.getResourceEntryName(R.drawable.event_image));
+        if (uri == null) {
+            throw new IllegalStateException("Why is it null?");
+        }
```

Comment:
I would go for assertNotNull(uri) simply, but ok

Reply:
It was just for the test, no big deal